src/ring_theory/graded_algebra/radical.lean

```diff
@@ -37,23 +37,24 @@ variables {ι R A : Type*}
 variables [comm_semiring R] [comm_ring A] [algebra R A]
 variables [linear_ordered_cancel_add_comm_monoid ι] [decidable_eq ι]
 variables (𝒜 : ι → submodule R A) [graded_algebra 𝒜]
-variable [Π (I : homogeneous_ideal 𝒜) (x : A),
-  decidable_pred (λ (i : ι), graded_algebra.proj 𝒜 i x ∉ I)]
-variable [Π (i : ι) (x : 𝒜 i), decidable (x ≠ 0)]
```

Comment:
I'm a little concerned that the linter didn't complain about these! Either way, they're only used by the proofs not the statements, so I removed them.